js/okx.js

```diff
@@ -2711,23 +2714,28 @@ module.exports = class okx extends Exchange {
         //       "selected": true
         //     }
         //
+        //      {
+        //          "chain":"USDT-Polygon",
+        //          "ctAddr":"",
+        //          "ccy":"USDT",
+        //          "to":"6",
+        //          "addr":"0x1903441e386cc49d937f6302955b5feb4286dcfa",
+        //          "selected":true
+        //      }
         const address = this.safeString (depositAddress, 'addr');
         let tag = this.safeString2 (depositAddress, 'tag', 'pmtId');
         tag = this.safeString (depositAddress, 'memo', tag);
         const currencyId = this.safeString (depositAddress, 'ccy');
         currency = this.safeCurrency (currencyId, currency);
         const code = currency['code'];
         const chain = this.safeString (depositAddress, 'chain');
-        const networks = this.safeValue (currency, 'networks', {});
```

Comment:
I'm afraid, this change will break USDT ERC20 and USDT TRC20 (TRX), and probably other currencies as well, and will cause de-unification rather than more unification. Please compare the following with and without the edits:

```
node examples/js/cli okx fetchDepositAddress USDT '{"network":"ERC20"}'
node examples/js/cli okx fetchDepositAddress USDT '{"network":"ETH"}'  # should be the same as the one above

node examples/js/cli okx fetchDepositAddress USDT '{"network":"TRC20"}'
node examples/js/cli okx fetchDepositAddress USDT '{"network":"TRX"}'  # should be the same as the one above

node examples/js/cli okx fetchDepositAddress USDT '{"network":"Omni"}'
node examples/js/cli okx fetchDepositAddress BTC
node examples/js/cli okx fetchDepositAddress ETH
node examples/js/cli okx fetchDepositAddress USDT
```

Reply:
okay sure thing, sorry I overlooked the TRX and ETH network naming types, will re-write this